alma/views/js/admin/alma.js

```diff
@@ -23,6 +23,25 @@
 
 (function ($) {
     $(function () {
+        if ($('.alma.share-of-checkout').length > 0) {
+            $('.btn-share-of-checkout').on('click', function() {
+                $.ajax({
+                    type: 'POST',
+                    url: 'ajax-tab.php',
+                    dataType: 'json',
+                    data: {
+                        ajax: true,
+                        controller: 'AdminAlmaShareOfCheckout',
+                        action: 'SettingShareOfCheckout',
+                        token: $(this).attr('data-token'),
+                        share: $(this).attr('data-share')
+                    },
+                    async   : true,
+                    cache   : false
+                });
```

Comment:
catch success & failure on ajax call and hide popin only if success

if failure => display default error message (`Impossible to save the Share of Checkout settings, please try again later `)